gym/lib/gym/generators/package_command_generator_legacy.rb

```diff
@@ -40,6 +42,15 @@ def pipe
         [""]
       end
 
+      # Place where the IPA file will be created, so it can be safely moved to the destination folder
+      def temporary_output_path
+        unless Gym.cache[:temporary_output_path]
+          Gym.cache[:temporary_output_path] = "#{Tempfile.new('gym').path}.gym_output"
```

Comment:
Oh, nice catch. Please change it at both locations, as this is an un-necessary string adding :+1: